admin/services/data-access/pupil.data.service.js

```diff
@@ -171,10 +171,10 @@ pupilDataService.sqlFindPupilsByDfeNumber = async function (dfeNumber, sortDirec
       sortBy = 'lastName'
   }
   const sql = `
-      SELECT 
-        p.*    
+      SELECT p.*, g.group_id 
       FROM ${sqlService.adminSchema}.${table} p 
       INNER JOIN school s ON s.id = p.school_id
+      LEFT JOIN ${sqlService.adminSchema}.[pupilGroup] g ON p.id = g.pupil_id
```

Comment:
just one question here: How is the query behaving if you don't have an assigned group?

Reply:
thats what the LEFT JOIN is for

Reply:
@GeorgeChatzigiannis the left join ensures we still get a successful result, if there are no pupils assigned to groups you'll get a group_id column with null values.

Reply:
@pris54 ok just wanted to confirm if it was tested and working as intended basically